README.md

```diff
@@ -1,4 +1,4 @@
-# Jamaican Bobsleigh Team
+# Jamaican Bobsleigh Team [![Build Status](https://travis-ci.com/RobinStephenson/jbt.svg?token=HjdBafwzwwqM31btobaT&branch=master)](https://travis-ci.com/RobinStephenson/jbt)
```

Comment:
This link is broken so remove this.
As the project is public we have to use travis-cs.**ORG** you can copy over our travis settings onto there if you want to